lib/rules/selector-max-specificity/README.md

```diff
@@ -48,7 +48,7 @@ The following patterns are considered problems:
 ```css
 .foo {
   color: red;
-  @nest .baz .bar & {
```

Comment:
`@nest` doesn't exist anymore.

I don't mind splitting this out into a separate PR.

Reply:
It was part of the a specification at one time. Let's not touch it for now.

Reply:
let's keep both for now, to make the change easier to review :)